src/Tribe/Service_Providers/Context.php

```diff
@@ -270,6 +270,38 @@ static function ( $post_type ) {
 					Tribe__Context::REQUEST_VAR => [ TEC::TAXONOMY ],
 				],
 			],
+			'view_url' => [
+				'read'  => [
+					Tribe__Context::FUNC => [
+						static function () {
+							return Utils\View::get_data( 'url', Tribe__Context::NOT_FOUND );
+						},
+					],
+					Tribe__Context::LOCATION_FUNC => [
+						// Handles the datepicker request.
+						'view_data',
+						static function ( $data ) {
+							$date_k = 'tribe-bar-date';
+							return is_array( $data ) && isset($data[ $date_k ])
+								? add_query_arg( [ $date_k => $data[ $date_k ]], tribe_get_request_var( 'url', home_url() ) )
+								: Tribe__Context::NOT_FOUND;
+						}
```

Comment:
* :no_entry_sign: Comma required after last value in array declaration
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 3b71c3f103a8072ae848e31d86b47c24</sub>